airflow/www/decorators.py

```diff
@@ -101,8 +101,10 @@ def has_dag_access(**dag_kwargs):
     def decorator(f):
         @functools.wraps(f)
         def wrapper(self, *args, **kwargs):
+            request_arguments = request.args.to_dict()
```

Comment:
```suggestion
            request_arguments = request.values.to_dict()
```

Exists in Flask and already does this merging.

Reply:
http://flask.pocoo.org/docs/1.0/api/#flask.Request.values